bin/reth/src/cli/mod.rs

```diff
@@ -31,6 +31,19 @@ pub struct Cli<Ext: RethCliExt = ()> {
     #[clap(subcommand)]
     command: Commands<Ext>,
 
+    /// Add a secondary node
+    ///
+    /// Configures the ports of the node to avoid conflicts with the defaults.
+    /// This is useful for running multiple nodes on the same machine.
+    ///
+    /// Changes to the following port numbers:
+    /// - DISCOVERY_PORT: 30304
+    /// - AUTH_PORT: 8552
+    /// - HTTP_RPC_PORT: 8544
+    /// - WS_RPC_PORT: 8547
+    #[arg(long, global = true)]
+    secondary: bool,
```

Comment:
Imo it would make more sense to have a sort of flag that would increment the ports by a given amount. That would allow you to run 2+ on the same machine. Not sure what the flag would be called, but something like `--instance` maybe? E.g. `--instance 1` uses `8545` and so on, `--instance 2` uses `8548` (has to increment by 2 because of websockets)